admin/spam/views.py

```diff
@@ -126,23 +159,18 @@ def get_queryset(self):
         return Comment.find(query).sort(self.ordering)
 
     def get_context_data(self, **kwargs):
-        queryset = kwargs.pop('object_list', self.object_list)
```

Comment:
An alternative would be to skip ODM entirely and build something that talks to the API but acts like a django queryset. I have one of these things. It would have to be modified to be able to write and to work with our api but it wouldn't be hard to modify.